assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -231,7 +231,7 @@ const MediaElement = ({
       ref={ref}
       data-testid="mediaElement"
       data-id={resourceId}
-      className={'mediaElement'}
+      className={`mediaElement mediaElement${type}`}
```

Comment:
Let's add the type to the `data-testid` instead, and perhaps add a dash (or alternatively properly camel-case it):

```js
data-testid={`mediaElement-${type}`}
```

There are only 2 Karma tests currently referencing `[data-testid="mediaElement"]`, which can be easily updated

Reply:
There are also 3 references in the e2e tests. But I am happy to change it.

Reply:
This changed in ALOT of places. But the work is done now. https://github.com/google/web-stories-wp/pull/5353/commits/8a0b2d00595a2b5e269742b92e4fc69fd6e83aeb